### DISCUSSION TITLE
UnoCSS support

### DISCUSSION THREAD

**Discussion by andrewdoro** [heroui-inc/tailwind-variants#40] [Ideas]
What would it take to support UnoCSS, it has a similar syntax to tailwind.
https://unocss.dev/

**Comment by omarkhatibgg** [comment]
I landed on `tailwind variants` from unocss.

I was more interested in uno attributify mode but this would be bad for a design system since overriding a class is impossible.

Unocss has many presets so I assume it would work only if you use Tailwind preset and in class mode only, as the underlying plugin `tailwind-merge` solve the conflicts here.

**Reply by its-monotype** [reply]
Where I can check info about "tailwind variants" from UnoCSS, can you provide me with a link, please 🙏

**Reply by omarkhatibgg** [reply]
https://unocss.dev/

**Comment by zhondori** [comment]
Hello. I tried to use it with @unocss/unocss and it worked.

[View on stackblitz](stackblitz.com/edit/vitejs-vite-mmenhd?file=src/App.vue)

But I don't know about responsive variants yet

**Reply by roonie007** [reply]
@zhondori the stackblitz link is incorrect
